src/components/AnnualReport/ARDate.vue

```diff
@@ -12,18 +12,19 @@
 <script lang="ts">
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
-import { State, Getter } from 'vuex-class'
+import { Getter } from 'pinia-class'
 import { DateMixin } from '@/mixins'
+import { useBusinessStore } from '@/stores/businessStore'
+import { useRootStore } from '@/stores/rootStore'
 
 @Component({
   mixins: [DateMixin]
 })
 export default class ArDate extends Vue {
```

Comment:
Sev this works! I was thinking to make this change in the other ticket since there is a checkbox that mention it? https://github.com/bcgov/entity/issues/16207

Reply:
Sure.